### PR TITLE
rename persistenceSDK to persistence-sdk

### PR DESCRIPTION
**Pull request by Anmol1696**
## 1. Overview

Rename persistenceSDK to persistence-sdk

### CODE REVIEW

**Review by puneet2019 — DISMISSED** [review]
LGTM, can you re-run the proto-definitions?

**Review by snj33v — APPROVED** [review]

File: scripts/protocgen.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    --gocosmos_out=plugins=interfacetype+grpc,\
+Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
+    --grpc-gateway_out=logtostderr=true,allow_colon_final_segments=true:. \
+  $(find "${dir}" -maxdepth 1 -name '*.proto')
+
+done
+
+# move proto files to the right places
+cp -r github.com/persistenceOne/persistence-sdk/* ./
+rm -rf github.com
```
Comment (ajansari95) [inline scripts/protocgen.sh:30]: newline

File: simapp/genesis_account_test.go [hunk 2]
```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
-	"github.com/persistenceOne/persistenceSDK/simapp"
+	"github.com/persistenceOne/persistence-sdk/simapp"
 
 	"github.com/stretchr/testify/require"
 	"github.com/tendermint/tendermint/crypto"
```
Comment (ajansari95) [inline simapp/genesis_account_test.go:13]: grouping of imports

File: scripts/protocgen.sh (hunk 1 above)
Comment (puneet2019) [inline scripts/protocgen.sh:30]:
```suggestion
rm -rf github.com

```

**Review by puneet2019 — APPROVED** [review]

**Review by puneet2019 — APPROVED** [review]

**Pull request merged**